package.json

```diff
@@ -1,63 +1,63 @@
 {
-  "name": "@lokalise/frontend-http-client",
-  "version": "2.1.0",
-  "description": "Opinionated HTTP client for the frontend",
-  "files": ["dist/**", "LICENSE", "README.md"],
-  "main": "./dist/index.cjs",
-  "types": "./dist/index.d.ts",
-  "module": "./dist/index.mjs",
-  "type": "module",
-  "exports": {
-    ".": {
-      "import": {
-        "types": "./dist/index.d.ts",
-        "default": "./dist/index.mjs"
-      },
-      "require": {
-        "types": "./dist/index.d.cts",
-        "default": "./dist/index.cjs"
-      }
-    }
-  },
-  "author": {
-    "name": "Lokalise",
-    "url": "https://lokalise.com/"
-  },
-  "homepage": "https://github.com/lokalise/frontend-http-client",
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/lokalise/frontend-http-client.git"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "scripts": {
-    "build:dev": "tsc",
-    "build:release": "tsup",
-    "clean": "rimraf dist",
-    "lint": "biome check . && tsc --project tsconfig.lint.json --noEmit",
-    "lint:fix": "biome check --write",
-    "test": "vitest run --coverage",
-    "prepublishOnly": "npm run clean && npm run build:release"
-  },
-  "dependencies": {
-    "fast-querystring": "^1.1.2"
-  },
-  "peerDependencies": {
-    "wretch": "^2.8.0",
-    "zod": "^3.22.0"
-  },
-  "devDependencies": {
-    "@biomejs/biome": "^1.8.3",
-    "@lokalise/biome-config": "^1.0.0",
-    "@types/node": "^22.0.0",
-    "@vitest/coverage-v8": "^2.0.1",
-    "jest-fail-on-console": "^3.1.2",
-    "mockttp": "^3.13.0",
-    "rimraf": "^6.0.0",
-    "tsup": "8.3.5",
-    "typescript": "~5.7.2",
-    "vitest": "^2.0.1"
-  },
-  "keywords": ["frontend", "web", "browser", "http", "client", "zod", "validation", "typesafe"]
+    "name": "@lokalise/frontend-http-client",
```

Comment:
had to do reformatting, as library repos do not use package-lock file, so CI picked up the latest version of biome and CI broke